data-model/src/main/java/org/alfresco/repo/search/impl/querymodel/impl/functions/FTSPhrase.java

```diff
@@ -47,6 +47,8 @@
     public final static String ARG_PHRASE = "Phrase";
 
     public final static String ARG_PROPERTY = "Property";
+
+    public final static String ARG_IS_PHRASE = "IsPhrase";
```

Comment:
This name clashes with ARG_PHRASE. Please change it or add a clear JavaDoc explaining what it means.

Reply:
What do you think about ARG_HAS_QUOTATION_MARKS? Meanwhile, I will add a JavaDoc.

Reply:
I'm not sure, perhaps you can use a short version ARG_HAS_QUOTES.